### ISSUE TITLE
Annotator is broken in all projects.

### ISSUE THREAD

**Issue by gfr10598** [m-lab/annotation-service#57]
Annotator is failing with Bad Gateway
http://status.mlab-oti.measurementlab.net:3000/dashboard/db/ndt-pipeline?refresh=1m&orgId=1&panelId=29&fullscreen&from=1520520330311&to=1520531130312&var-interval=10m

Looks like they have added a column to the csv, which is breaking the import of new data.
A  2018/03/08 07:17:00 data.go:35: Corrupted Data: wrong number of columns 
A  2018/03/08 07:17:00 geo-g2.go:101: [18918 en EU Europe CY Cyprus 04 Ammochostos   Protaras  Asia/Famagusta 1] 
A  2018/03/08 07:17:00 geo-g2.go:100: Incorrect number of columns in IP list got:  14  wanted:  13
A  2018/03/08 07:16:59 data.go:35: Corrupted Data: wrong number of columns
A  2018/03/08 07:16:59 geo-g2.go:101: [18918 en EU Europe CY Cyprus 04 Ammochostos   Protaras  Asia/Famagusta 1]
A  2018/03/08 07:16:59 geo-g2.go:100: Incorrect number of columns in IP list got:  14  wanted:  13
A  2018/03/08 07:16:58 main.go:13: Beginning Setup

**Comment by stephen-soltesz** [comment]
Long term, it could be helpful to refer to columns by name using a map interface like https://github.com/recursionpharma/go-csv-map rather than by column number, which is more brittle.

**Comment by stephen-soltesz** [comment]
The headers for City Locations have changed between Feb & March.

```
$ diff <( head -1 GeoLite2-City-CSV_20180206/GeoLite2-City-Locations-en.csv ) <( head -1 GeoLite2-City-CSV_20180306/GeoLite2-City-Locations-en.csv )
1c1
< geoname_id,locale_code,continent_code,continent_name,country_iso_code,country_name,subdivision_1_iso_code,subdivision_1_name,subdivision_2_iso_code,subdivision_2_name,city_name,metro_code,time_zone
---
> geoname_id,locale_code,continent_code,continent_name,country_iso_code,country_name,subdivision_1_iso_code,subdivision_1_name,subdivision_2_iso_code,subdivision_2_name,city_name,metro_code,time_zone,is_in_european_union
```

**Comment by gfr10598** [comment]
Fixed in #59

**Comment by stephen-soltesz** [comment]
Today's fixes are now in prod. https://travis-ci.org/m-lab/annotation-service/builds/351098166

The annotator error rates are back down to zero.
